config/bugsnag.php

```diff
@@ -99,7 +99,6 @@
     */
 
     'proxy' => array_filter([
-        'http' => env('HTTP_PROXY'),
```

Comment:
Restored in https://github.com/bugsnag/bugsnag-laravel/pull/145 since it's still safe for cli, so no real reason to remove it.